Client/Ecosia/UI/WhatsNew/WhatsNewCell.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Core
+
+final class WhatsNewCell: UITableViewCell {
+    
+    static let reuseIdentifier = String(describing: WhatsNewCell.self)
```

Comment:
There is a `ReusableCell` protocol that does exactly this for the cell (with the difference it instead called `cellIdentifier`). No blocker, just letting you know in case you prefer using it.

Reply:
Thanks for reminder will definitely update to remove that! 🚀